.eslintrc.js

```diff
@@ -21,15 +21,12 @@ module.exports = {
 	plugins: ["react", "jest", "jsx-a11y", "react-hooks", "sort-exports"],
 	rules: {
 		"global-require": "off",
-		indent: ["error", "tab"],
 		"no-tabs": [
 			"error",
 			{
 				allowIndentationTabs: true,
 			},
 		],
-		"react/jsx-indent": [2, "tab"],
```

Comment:
this was done because there were conflicts with prettier and eslint: 

before 

conflicting eslint and prettier 

howardj@arc-20750 arc-themes-components % npx eslint-config-prettier .eslintrc.js 
The following rules are unnecessary or might conflict with Prettier:

- indent
- react/jsx-indent
- react/jsx-indent-props
howardj@arc-20750 arc-themes-components % 

after 

howardj@arc-20750 arc-themes-components % npx eslint-config-prettier .eslintrc.js
No rules that are unnecessary or conflict with Prettier were found.